### DISCUSSION TITLE
About copying images in Storage

### DISCUSSION THREAD

**Discussion by AkiUnleash** [supabase/supabase#4006] [Questions] [answered]
Hi!  I have a question.

I want to copy the image in supabase storage.
There seems to be an API, but is there a library?
https://supabase.github.io/storage-api/#/object/post_object_copy

I was able to find move, but I can't find copy.
https://supabase.com/docs/reference/javascript/storage-from-move

I hope you can tell me about it.

**Comment by inian** [comment, marked as answer]
Created an issue here @AkiUnleash https://github.com/supabase/storage-js/issues/29
